recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0" %}
+{% set git_branch = "master" %}
+{% set build_number = "0" %}
+
+package:
+  name: ale
+  version: {{ version }}
+
+source:
+  git_url: 'https://github.com/USGS-Astrogeology/ale.git'
+  git_branch: {{ git_branch }}
+
+build:
+  number: {{ build_number }}
+
+# Shotgun strat on requirements until we can narrow them down
+requirements:
+  build:
+  - python=3.7.1
```

Comment:
Can we unpin the python version?

Reply:
It appears to be building properly without this pinned. Will confirm when I can test the uploading and installing process.